src/components/profiles/Profile.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
```

Comment:
@a-braham What is the essence of this file.

Reply:
Done